src/processors/vault/vault.ts

```diff
@@ -112,6 +114,15 @@ const getLatestVault = async (
   )
   let vault = current
   if (!vault) {
+    result.promises.push(
+      ensureExchangeRates(ctx, block, [
+        ['ETH', 'USD'],
```

Comment:
nit: These could be constants as well. Same for other processors and their related files

Reply:
I am not sure what you mean.